src/app/components/normal-component/lobby/lobby.component.html

```diff
@@ -33,7 +33,9 @@
                     <td class="td">{{ part.data.playerZero }}</td>
                     <td class="td">
                         <ng-container *ngIf="part.data.playerOne else waitingForOpponent">{{ part.data.playerOne }}</ng-container>
-                        <ng-template #waitingForOpponent i18n>Waiting for opponent</ng-template>
+                        <ng-template #waitingForOpponent>
+                            <ng-container i18n>Waiting for opponent</ng-container>
+                        </ng-template>
```

Comment:
waarom?

Reply:
fiou, bonne question :D ?
Tu veux que je revert ?

Reply:
ptet pour simplifier

Reply:
Ah, j'ai revert et bizarrement le linter se plaint que l'absence d'attribut i18n sur les ng-template mais pas sur les ng-container!

Reply:
et si tu mets l'attribut i18n sur le ng-template ? Enfin me semble qu'il y a rien à se plaindre sur comment c'était fait précédemment right? `<ng-template #waitingForOpponent i18n>Waiting for opponent</ng-template>`